Packs/Gurucul/pack_metadata.json

```
@@ -0,0 +1,27 @@
+{
+    "name": "Gurucul Risk Analytics",
+    "description": "Gurucul Risk Analytics (GRA) is a Unified Security and Risk Analytics platform.",
+    "support": "partner",
+    "currentVersion": "1.0.0",
+    "author": "Gurucul",
+    "url": "https://www.gurucul.com",
+    "email": "support@gurucul.com",
+    "githubUser": "graadmin",
+    "created": "2020-12-02T18:00:00Z",
+    "categories": [
+        "Analytics & SIEM"
```

I believe the type of the pack/integration should be `Data Enrichment & Threat Intelligence`.
See: https://xsoar.pan.dev/docs/concepts/use-cases#data-enrichment--threat-intelligence
```suggestion
        "Data Enrichment & Threat Intelligence"
```